bundles/binding/org.openhab.binding.zwave/database/philio/pat02c.xml

```diff
@@ -0,0 +1,133 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Product>
+    <Model>PAT02-C</Model>
+    <Label lang="en">Flood Sensor</Label>
+    <CommandClasses>
+        <Class><id>0x20</id></Class>    <!-- COMMAND_CLASS_BASIC -->
+        <Class><id>0x30</id></Class>    <!-- COMMAND_CLASS_SENSOR_BINARY_V2 -->
+        <Class><id>0x31</id></Class>    <!-- COMMAND_CLASS_SENSOR_MULTILEVEL_V5 -->
+        <Class><id>0x59</id></Class>    <!-- COMMAND_CLASS_ASSOCIATION_GRP_INFO -->
+        <Class><id>0x5A</id></Class>    <!-- COMMAND_CLASS_DEVICE_RESET_LOCALLY -->
+        <Class><id>0x5E</id></Class>    <!-- COMMAND_CLASS_ZWAVEPLUS_INFO_V2 -->
+        <Class><id>0x70</id></Class>    <!-- COMMAND_CLASS_CONFIGURATION -->
+        <Class><id>0x71</id></Class>    <!-- COMMAND_CLASS_NOTIFICATION_V4 -->
+        <Class><id>0x72</id></Class>    <!-- COMMAND_CLASS_MANUFACTURER_SPECIFIC_V2 --> 
+        <Class><id>0x73</id></Class>    <!-- COMMAND_CLASS_POWERLEVEL -->
+        <Class><id>0x7A</id></Class>    <!-- COMMAND_CLASS_FIRMWARE_UPDATE_MD_V2 -->
+        <Class><id>0x80</id></Class>    <!-- COMMAND_CLASS_BATTERY -->
+        <Class><id>0x84</id></Class>    <!-- COMMAND_CLASS_WAKE_UP_V2 -->
+        <Class><id>0x85</id></Class>    <!-- COMMAND_CLASS_ASSOCIATION_V2-->
+        <Class><id>0x86</id></Class>    <!-- COMMAND_CLASS_VERSION_V2 -->
+        <Class><id>0x8F</id></Class>    <!-- COMMAND_CLASS_MULTI_CMD -->
+        <Class><id>0x98</id></Class>    <!-- COMMAND_CLASS_SECURITY -->
+    </CommandClasses>
+    <Configuration>
+        <Parameter>
+            <Index>1</Index>
+            <Type>byte</Type>
+            <Default>0</Default>
+            <Minimum>0</Minimum>
+            <Maximum>1</Maximum>
+            <Size>1</Size>
+            <Label lang="en">Basic Set OFF Level</Label>
+            <Help lang="en"><![CDATA[Setting the BASIC command value<br/>
+                <ul><li>When the flood trigger off(0x00), send the BASIC CC to the group 2</li></ul>]]></Help>
+        </Parameter>
+        <Parameter>
+            <Index>2</Index>
+            <Type>byte</Type>
+            <Default>-1</Default>
```

Comment:
I think I would be tempted us use 255 rather than -1 in these two places. I know in theory it's a signed value (as per the spec) but in practice, it tends to be used unsigned.